File: Temperature Conversion/C - F/C2F.java
import java.util.Scanner;
class C2F{
    public static void main(String[] args){
        Scanner sc=new Scanner(System.in);
        System.out.print("Enter Temperature (in Celcius) : ");
        double c=sc.nextDouble();
        sc.close();
        double f=((c/5.0)*9)+32.0;
        System.out.print("Temperature in Fahrenheit : "+f);
    }
}

File: Problem Statements/Temperature Conversion/C - F/C2F.c
#include<stdio.h>
void main(){
    double c,f=0.0;
    printf("Enter Temperature (in Celcius) : ");
    scanf("%lf",&c);
    f=((c/5.0)*9)+32.0;
    printf("Temperature in Fahrenheit : %0.2f",f);
}